examples/webrx/.gitignore

```diff
@@ -0,0 +1,2 @@
+obj
+bin
```

Comment:
please take a look at other gitignores. it should include all module files we don't need (and those should be removed from the commit)